Source/WebCore/style/values/grid/StyleGridTrackSize.h

```diff
@@ -142,10 +224,19 @@ struct GridTrackSize {
     bool hasFixedMaxTrackBreadth() const { return m_maxTrackBreadthIsFixed; }
 
 private:
-    GridTrackSizeType m_type;
-    Style::GridTrackBreadth m_minTrackBreadth;
-    Style::GridTrackBreadth m_maxTrackBreadth;
-    Style::GridTrackBreadth m_fitContentTrackBreadth;
+    friend struct Blending<GridTrackSize>;
+
+    enum class Type : uint8_t {
+        Breadth,
+        MinMax,
+        FitContent
+    };
+    Type type() const { return m_type; }
+
+    Type m_type;
+    GridTrackBreadth m_minTrackBreadth;
+    GridTrackBreadth m_maxTrackBreadth;
+    GridTrackFitContentLength m_fitContentTrackLength;
```

Comment:
Seems good style to always initialize as long as it doesn’t have any cost. Can then remove constructor initializations that set members to those defaults.
```suggestion
    Type m_type { Type::Breadth };
    GridTrackBreadth m_minTrackBreadth { CSS::Keyword::Auto { } };
    GridTrackBreadth m_maxTrackBreadth { CSS::Keyword::Auto { } };
    GridTrackFitContentLength m_fitContentTrackLength { 0_css_px };
```